anisoap/representations/ellipsoidal_density_projection.py

```diff
@@ -76,7 +70,7 @@ def pairwise_ellip_expansion(
 
     """
     tensorblock_list = []
-    keys = np.array(neighbor_list.keys.asarray(), dtype=int)
+    keys = np.asarray(neighbor_list.keys, dtype=int)
```

Comment:
a lot of these api changes present errors for my old version of rascaline, I'm reinstalling right now to see if it works.

Reply:
Great, installing the latest version of rascaline fixed my errors